pkgs/servers/sql/mysql/jdbc/default.nix

```
@@ -1,18 +1,32 @@
-{lib, stdenv, fetchurl, ant, unzip}:
+{ lib, stdenv, fetchurl, ant, unzip }:
 
 stdenv.mkDerivation rec {
-  name = "mysql-connector-java-5.1.46";
-  builder = ./builder.sh;
+  pname = "mysql-connector-java";
+  version = "5.1.46";
 
   src = fetchurl {
-    url = "http://dev.mysql.com/get/Downloads/Connector-J/${name}.zip";
+    url = "http://dev.mysql.com/get/Downloads/Connector-J/mysql-connector-java-${version}.zip";
```

Let's use https and pname

```suggestion
    url = "https://dev.mysql.com/get/Downloads/Connector-J/${pname}-${version}.zip";
```